randovania/data/json_data/prime1.txt

```diff
@@ -1279,379 +2367,822 @@ Asset id: 1125030300
+
+> Event (Save Station A Barrier Opened); Heals? False
+  > Dock to Main Quarry
+      Trivial
 
 ----------------
 Security Access A
 Asset id: 3345300114
-> Dock to Main Quarry; Heals? False
-  * Normal Door to Main Quarry/Dock to Security Access A
+> Dock to Main Quarry; Heals? False; Spawn Point
+  * Ice Door to Main Quarry/Dock to Security Access A
+  > Dock to Mine Security Station
+      Trivial
+  > Pickup (Missile)
+      All of the following:
+          Power Bomb and Morph Ball
+          X-Ray Visor or Invisible Objects (Beginner)
 
 > Dock to Mine Security Station; Heals? False
-  * Normal Door to Mine Security Station/Dock to Security Access A
+  * Ice Door to Mine Security Station/Dock to Security Access A
+  > Dock to Main Quarry
+      Trivial
+
+> Pickup (Missile); Heals? False
+  * Pickup 74; Major Location? False
+  > Dock to Main Quarry
+      Trivial
 
 ----------------
 Ore Processing
 Asset id: 2547167990
 > Dock to Research Access; Heals? False
-  * Normal Door to Research Access/Dock to Ore Processing
+  * Ice Door to Research Access/Dock to Ore Processing
+  > Dock to Elevator Access A
+      Morph Ball Bomb and Morph Ball and Spider Ball
 
 > Dock to Storage Depot B; Heals? False
-  * Normal Door to Storage Depot B/Dock to Ore Processing
+  * Ice Door to Storage Depot B/Dock to Ore Processing
+  > Dock to Waste Disposal
+      Grapple Beam
+  > Dock to Elevator Access A
+      Trivial
 
 > Dock to Waste Disposal; Heals? False
-  * Normal Door to Waste Disposal/Dock to Ore Processing
+  * Ice Door to Waste Disposal/Dock to Ore Processing
+  > Dock to Storage Depot B
+      Grapple Beam
+  > Dock to Elevator Access A
+      Trivial
 
-> Dock to Elevator Access A; Heals? False
-  * Normal Door to Elevator Access A/Dock to Ore Processing
+> Dock to Elevator Access A; Heals? False; Spawn Point
+  * Ice Door to Elevator Access A/Dock to Ore Processing
+  > Dock to Research Access
+      Trivial
+  > Dock to Storage Depot B
+      Morph Ball Bomb and Power Bomb and Space Jump Boots and Morph Ball and Spider Ball
 
 ----------------
 Mine Security Station
 Asset id: 2507085138
-> Dock to Security Access A; Heals? False
-  * Normal Door to Security Access A/Dock to Mine Security Station
+> Dock to Security Access A; Heals? False; Spawn Point
+  * Ice Door to Security Access A/Dock to Mine Security Station
+  > Dock to Security Access B
+      Wave Beam
+  > Dock to Storage Depot A
+      Wave Beam and Scan Visor and Power Bomb and Morph Ball
 
 > Dock to Security Access B; Heals? False
-  * Normal Door to Security Access B/Dock to Mine Security Station
+  * Wave Door to Security Access B/Dock to Mine Security Station
+  > Dock to Security Access A
+      Wave Beam
 
 > Dock to Storage Depot A; Heals? False
-  * Normal Door to Storage Depot A/Dock to Mine Security Station
+  * Plasma Door to Storage Depot A/Dock to Mine Security Station
+  > Dock to Security Access A
+      Wave Beam
```

Comment:
- Add a new event called "Mine Security Station Barrier"
- Add a new node to this room for the event
- Connect the event to Security Access B door as Trivial
- Connect Security Access B door to the event with Scan Visor and Power Bomb
- Connect Security Access B door to Storage Depot A door with "After Barrier" event
- Connect Storage Depot A door to Security Access B door with "After Barrier" event
- Remove connection from Security Access A door to Storage Depot A door
- Remove connection from Storage Depot A door to Security Access A door